config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml

```diff
@@ -40,7 +40,7 @@ periodics:
           memory: "16Gi"
 
 # This is a sig-release-master-blocking job.
-- cron: '1 8 * * *' # Run daily at 00:01PST (8:01 UTC)
+- cron: '1 13 * * *' # Run daily at 00:05PST (13:01 UTC)
```

Comment:
I would vote for sth like 5pm UTC - this would mean that this would finish by 8am CET so we can start looking immediately after being in the office and it increases amount of time we have to debug and fix within a single day.